web/template/partial/stream/chat/tooltip-bottom.gohtml

```diff
@@ -0,0 +1,13 @@
+{{define "tooltip-bottom"}}
```

Comment:
Some tooltip components are super neat, I can't find where they are used though :thinking:

Reply:
Thats because they aren't used right now 😅

Reply:
Cool, hope I remember them the next time I need them :D